### PR TITLE
Fixes #2621. Fixes #2622

### LINKED ISSUE #2621: HI: check if the "double scaling" of Q_liver is correct
s. the discussion https://github.com/Open-Systems-Pharmacology/PK-Sim/pull/2620/files?w=1#r1194943507

### LINKED ISSUE #2622: HI: Hematocrit scaling factors below do not match the paper?
Hematocrit scaling factors below do not match the paper?
`private static readonly HIFactors _hematocritScalingFactor = createFactors(0.92, 0.88, 0.83);`

grafik
grafik

_Originally posted by @Yuri05 in https://github.com/Open-Systems-Pharmacology/PK-Sim/pull/2620#discussion_r1195522553_

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Core/Services/HIDiseaseStateImplementation.cs
Comment (Yuri05) [inline src/PKSim.Core/Services/HIDiseaseStateImplementation.cs:1]: As far as I understood the discussion in https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2621 the value origin for the blood flow factors must be cha ged now from Edginton to Johnson?

Comment (msevestre) [reply src/PKSim.Core/Services/HIDiseaseStateImplementation.cs:1]: Ha ha this is correct. Eagle eye Juri

File: src/PKSim.Core/Services/HIDiseaseStateImplementation.cs [hunk 1]
```diff
@@ -36,14 +36,14 @@ public static class ChildPughScore
...
-      private static readonly HIFactors _hepaticFlowScalingFactor = createFactors(1.3, 2.3, 3.4);
+      private static readonly HIFactors _hepaticFlowScalingFactor = createFactors(1.61, 2.34, 5.29);
       private static readonly HIFactors _hepaticVolumeScalingFactor = createFactors(0.69, 0.55, 0.28);
       private static readonly HIFactors _renalFlowScalingFactor = createFactors(0.88, 0.65, 0.48);
       private static readonly HIFactors _cardiacIndexScalingFactor = createFactors(1.11, 1.27, 1.36);
       private static readonly HIFactors _gfrScalingFactor = createFactors(1, 0.7, 0.36);
       private static readonly HIFactors _albuminScalingFactor = createFactors(0.81, 0.68, 0.5);
       private static readonly HIFactors _agpScalingFactor = createFactors(0.6, 0.56, 0.3);
-      private static readonly HIFactors _hematocritScalingFactor = createFactors(0.92, 0.88, 0.83);
+      private static readonly HIFactors _hematocritScalingFactor = createFactors(0.86, 0.822, 0.778);
```
Comment (Yuri05) [inline src/PKSim.Core/Services/HIDiseaseStateImplementation.cs:46]: In https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/2621#issuecomment-1552358781 HCT-Scalefactor for A is 0.866

**Comment by msevestre** [comment]
ok ready...merging

**Pull request merged**